_posts/2015-04-06-Simplified-Workspace-Creation.md

```diff
@@ -29,7 +29,7 @@ update your `~/.bazelrc` file.
 finds/generates, you can create a `tools/` directory in your project and
 Bazel will attempt to use that instead of the system-wide one.
 
-See the [getting started]({{ site_root }}/docs/getting-started.html) docs for more info about
+See the [getting started](https://docs.bazel.build/versions/master/getting-started.html) docs for more info about
```

Comment:
We have the `site.docs_site_url` defined in _config.yml to `https://docs.bazel.build`. Let's use that variable instead?